src/components/new-test/QuestionBlock.tsx

```diff
@@ -5,15 +5,22 @@ import { SuperSelect } from './SuperSelect';
 import { SuperInput } from './SuperInput';
 import { Timer } from './Timer';
 
+
 type QuestionBlockType = {
   value: string;
+  difficultyValue: string;
   handleTypeChange: (event: SelectChangeEvent) => void;
+  handleDifficultyChange: (event: SelectChangeEvent) => void;
```

Comment:
please, change naming here. props handlers should start with 'on' and internal handlers with 'handle'
here is the topic https://jaketrent.com/post/naming-event-handlers-react/

Reply:
fix naming